apps/base/utils.py

```diff
@@ -18,20 +17,17 @@ def paginated_queryset(queryset, request):
     return (paginator, result_page)
 
 
-@deconstructible
```

Comment:
I am actually not getting of the point of removing this `deconstrutible`. Can you please explain?